integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/CarbonLogReader.java

```diff
@@ -178,6 +182,7 @@ public boolean checkForLog(String expected, int timeout, int numberofOccurrences
             }
             TimeUnit.SECONDS.sleep(1);
         }
+        LOG.warn("Found "+ this.getNumberOfOccurencesForLog(expected) +" occurrences while expecting " + numberofOccurrences);
```

Comment:
In this case , I thought it would be better to print the whole content of the logs in within an identifiable string like 
===========
Whole logs
===========

so that we can manually see whether that content is there in the logs checked. Coz we could see them in logs downloaded but the missing part is not being  sure whether it is collected by log tailer.

Reply:
added the log to the same warn